assembl/static2/js/app/app.jsx

```diff
@@ -55,7 +76,37 @@ const mapDispatchToProps = dispatch => ({
   },
   addContext: (path, debateId, connectedUserId, connectedUserName) => {
     dispatch(addContext(path, debateId, connectedUserId, connectedUserName));
+  },
+  putTimelineInStore: (timeline) => {
```

Comment:
why do you need timeline in store ? I think you should just use the query each time you need it (or even better, put it in the new context API and use some sort of `<TimelineProvider>` component and we should do the same for everything we need in multiple different places.

Context API doc: https://reactjs.org/blog/2018/03/29/react-v-16-3.html

Reply:
Sorry, this is the real doc: https://reactjs.org/docs/context.html